logging/log_formatters.go

```diff
@@ -126,6 +106,14 @@ type AcraTextFormatter struct {
 	Hooks []FormatterHook
 }
 
+func (f *AcraTextFormatter) SetServiceName(serviceName string) {
+	// service name is ignored by plaintext formatter, so just do nothing
+}
```

Comment:
Oh... This is news to me. I thought it is mapped onto a log field or something, but now I've noticed that indeed, it's not really used in any way by the plaintext formatter.

By the way, is there a particular reason for why we have the “service name” thing at all, @Lagovas, @shadinua? Shouldn't the log connector be aware of the log source so that it's not necessary to set it? Or is it only because CEF has a mandatory field for that? Why JSON follows suit then?

Reply:
it's new for me too) yes, maybe we added it first when needed cef format. but anyway it's good to log source of logs especially when we have several running services to distinguish them